dbapi/src/main/java/com/griddynamics/jagger/dbapi/provider/SessionInfoProviderImpl.java

```diff
@@ -84,8 +87,19 @@ public void saveUserComment(Long sessionDataId, String userComment) throws Runti
         dataSaverService.saveUserComment(sessionDataId, userComment);
     }
 
-    public Long getTotalSize() throws RuntimeException {
-        return (Long) entityManager.createQuery("select count(sessionData.id) from SessionData as sessionData").getSingleResult();
+    public Long getTotalSize() {
+        Long result;
+        try {
+            result = (Long) entityManager.createQuery("select count(sessionData.id) from SessionData as sessionData").getSingleResult();
+        } catch (PersistenceException ex) {
+            Throwable rootCause = ExceptionUtils.getRootCause(ex);
+            if (rootCause != null && StringUtils.contains(rootCause.getMessage(), "doesn't exist")) {
+                result = 0L;
```

Comment:
Exception type is not enough to understand if this exception means that table doesn't exists.
Also, we use different vendors of DB: mysql and H2, so exception types are different. 
I don't think, that exception message for not existing tables will be changed in drivers... But if so, Exception type also can be changed, so this solution doesn't seem better to me.